### ISSUE TITLE
no output KML file of  roiginal gps data or calibrated gps data

### ISSUE THREAD

**Issue by DamonMIN** [iMorpheusAI/gpsCalibration#24]
Dear @Donnnnng @yangjiale1995 @likeopen @cuitaixiang @Morpheusai,
     It's a fantastic work. When I follow the instruction to run the program, there are no data files generated. When I run commands "./run.sh", some information are shown as following.
=====================================
WARNING: Package name "gpsCalibration" does not follow the naming conventions. It should start with a lower case letter and only contain lower case letters, digits and underscores.
... logging to /home/Amit/.ros/log/8f7b3fcc-f2b7-11e7-b1d2-6c0b8445aaa1/roslaunch-ubuntu-9452.log
Checking log directory for disk usage. This may take awhile.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

started roslaunch server http://ubuntu:53462/

SUMMARY
========

PARAMETERS
 * /rosdistro: indigo
 * /rosversion: 1.11.20

NODES
  /
    input_data_node (gpsCalibration/input_data_node)
    laserMapping (gpsCalibration/laserMapping)
    laserOdometry (gpsCalibration/laserOdometry)
    long_distance_track_process (gpsCalibration/long_distance_track_process_node)
    scanRegistration (gpsCalibration/scanRegistration)
    short_distance_track_process_node (gpsCalibration/short_distance_track_process_node)
    transformMaintenance (gpsCalibration/transformMaintenance)

ROS_MASTER_URI=http://localhost:11311

core service [/rosout] found
WARNING: Package name "gpsCalibration" does not follow the naming conventions. It should start with a lower case letter and only contain lower case letters, digits and underscores.
process[scanRegistration-1]: started with pid [9470]
process[laserOdometry-2]: started with pid [9471]
process[laserMapping-3]: started with pid [9472]
process[transformMaintenance-4]: started with pid [9473]
process[input_data_node-5]: started with pid [9474]
process[long_distance_track_process-6]: started with pid [9476]
process[short_distance_track_process_node-7]: started with pid [9486]
[scanRegistration-1] process has died [pid 9470, exit code -11, cmd /home/Amit/coding/gpsCalibration/devel/lib/gpsCalibration/scanRegistration __name:=scanRegistration __log:=/home/Amit/.ros/log/8f7b3fcc-f2b7-11e7-b1d2-6c0b8445aaa1/scanRegistration-1.log].
log file: /home/Amit/.ros/log/8f7b3fcc-f2b7-11e7-b1d2-6c0b8445aaa1/scanRegistration-1*.log
[laserOdometry-2] process has died [pid 9471, exit code -11, cmd /home/Amit/coding/gpsCalibration/devel/lib/gpsCalibration/laserOdometry __name:=laserOdometry __log:=/home/Amit/.ros/log/8f7b3fcc-f2b7-11e7-b1d2-6c0b8445aaa1/laserOdometry-2.log].
log file: /home/Amit/.ros/log/8f7b3fcc-f2b7-11e7-b1d2-6c0b8445aaa1/laserOdometry-2*.log
[laserMapping-3] process has died [pid 9472, exit code -11, cmd /home/Amit/coding/gpsCalibration/devel/lib/gpsCalibration/laserMapping __name:=laserMapping __log:=/home/Amit/.ros/log/8f7b3fcc-f2b7-11e7-b1d2-6c0b8445aaa1/laserMapping-3.log].
log file: /home/Amit/.ros/log/8f7b3fcc-f2b7-11e7-b1d2-6c0b8445aaa1/laserMapping-3*.log
Total Message Num Is 418

***********************publish message***********************
published 0 message,418 message left,maybe 7.0 minites to publish
=====================================

**Comment by Morpheusai** [comment]
DamonMIN, From your printing info, we have seen that there is something wrong in three processes: scanRegistration, laserOdometry and laserMapping. In our experiences, pls tell us your ubuntu version, pcl version. Tks for your attention.

**Comment by DamonMIN** [comment]
@Morpheusai 
Ubuntu version is 14.04, and pcl version is 1.7.2. Thank you for your help.

**Comment by likeopen** [comment]
hi all:
configure:

soure setup.sh
./run.sh
you need to wait a while 
output:

 
 
------------------ Original ------------------
From: "DamonMIN"; 
Date: 2018年1月8日(星期一) 上午10:49
To: "iMorpheusAI/gpsCalibration"; 
Cc: "likeopen"; "Mention"; 
Subject: Re: [iMorpheusAI/gpsCalibration] no output KML file of roiginal gpsdata or calibrated gps data (#24)

 

@Morpheusai
 Ubuntu version is 14.04, and pcl version is 1.7.2. Thank you for your help.

**Comment by DamonMIN** [comment]
@likeopen 
Of course, I wait the program run to completion. There are still no result data files generated.
==================================
published 400 message,18 message left,maybe 0.3 minites to publish
[INFO]In one cycle,send message:418,lost message:0,message loss rate < 0.01%
WARN: no GPS track,please check it
================================================================================REQUIRED process [long_distance_track_process-6] has died!
process has died [pid 14808, exit code 255, cmd /home/Amit/coding/gpsCalibration/devel/lib/gpsCalibration/long_distance_track_process_node ./data/original_gps_data.txt UTM 3 __name:=long_distance_track_process __log:=/home/Amit/.ros/log/8f7b3fcc-f2b7-11e7-b1d2-6c0b8445aaa1/long_distance_track_process-6.log].
log file: /home/Amit/.ros/log/8f7b3fcc-f2b7-11e7-b1d2-6c0b8445aaa1/long_distance_track_process-6*.log
Initiating shutdown!
================================================================================
[short_distance_track_process_node-7] killing on exit
[long_distance_track_process-6] killing on exit
[input_data_node-5] killing on exit
[transformMaintenance-4] killing on exit
==================================
Thank you for your suggestion.

**Comment by Morpheusai** [comment]
Hi, DamonMIN.
We have met this problem in ubuntu14.04.
pls  update your pcl-1.7.2 to pcl-1.8.3 and have a try.

**Comment by yangjiale1995** [comment]
“open ./data/original_data.txt error”.......obviously,  there is no original gps text in data directory. you should check data directory and make sure your original gps data text is exist also be named “original_data.txt”。
--
发自我的网易邮箱手机智能版


在 2018-01-13 14:47:00，DamonMIN  写道：

**Comment by Morpheusai** [comment]
Hi, DamonMIN:
     Make sure that you have put right original_data.txt in right folder ./data/.
     Tks for your suggesitons.
